### PR TITLE
kernel: package Broadcom BNX2X driver

### PR DESCRIPTION
**Pull request by ignisf**
bnx2x driver support for the x86 architecture. Includes module and firmware for Broadcom QLogic 5771x/578xx 10/20-Gigabit ethernet adapters.

(As discussed with @hauke and @KanjiMonster in IRC)

### CODE REVIEW

File: package/kernel/linux/modules/netdevices.mk [hunk 1]
```diff
@@ -991,3 +991,19 @@ define KernelPackage/bnx2/description
 endef
 
 $(eval $(call KernelPackage,bnx2))
+
+
+define KernelPackage/bnx2x
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=QLogic 5771x/578xx 10/20-Gigabit ethernet adapter driver
+  DEPENDS:=@PCI_SUPPORT +bnx2x-firmware +kmod-lib-crc32c +kmod-mdio +kmod-ptp
```
Comment (chunkeey) [inline package/kernel/linux/modules/netdevices.mk]: https://github.com/torvalds/linux/blob/master/drivers/net/ethernet/broadcom/Kconfig#L134

mentions zlib as well?

I've added the dependency and did some reshuffling. Please check.

https://git.openwrt.org/?p=openwrt/staging/chunkeey.git;a=commit;h=188ae12f39070d53c86d4f32fdb56dc40f9ea1c8

Comment (ignisf) [reply package/kernel/linux/modules/netdevices.mk]: will do, thanks

Comment (ignisf) [reply package/kernel/linux/modules/netdevices.mk]: @chunkeey, looks good to me :+1:

**Comment by chunkeey** [comment]
Merged. Thanks.

**Pull request closed without merging**